scss/main.scss

```diff
@@ -0,0 +1,9 @@
+@import './config';
+@import './normalize';
+
+body {
```

Comment:
I have a feeling you were going to put styles in another file, but just in case you missed it, would be great to have styles broken down into `base`, `components`, `pages`, or any metric you come up with. Primer css (the link i shared) does a good job with this.